gateway_to_backend/README.md

```diff
@@ -277,6 +277,55 @@ _Even if this version is increased, the API remains backward compatible.
 It just help backends development to identify if new features are
 present in a gateway._
 
+#### Set configuration data item
+
+Only processed if CDD_API feature flag is set on the gateway.
+
+- **Request:**
+
+    > **topic:** gw-request/set_configuration_data_item/*\<gw-id\>/\<sink-id\>*
+    >
+    > **content:** [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].SetConfigurationDataItemReq
+
+- **Response:**
+
+    > **topics:** gw-response/set_configuration_data_item/*\<gw-id\>/\<sink-id\>*
+    >
+    > **content:** [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].SetConfigurationDataItemResp
+    >
+    > **possible errors:**
+    > - INVALID_ROLE: Node is not a sink.
+    > - SINK_OUT_OF_MEMORY: Payload is too large or maximum number of items is reached.
+    > - INVALID_DEST_ENDPOINT: Endpoint is too large (it should be a 16 bit unsigned integer).
+    > - INVALID_DATA_PAYLOAD: Payload is rejected for this endpoint by the sink.
+    > - INVALID_SINK_ID: Invalid sink id in the request header.
+
+:warning:
+
+_Successfully setting an item must be treated as a configuration change on the
+sink, meaning that the gateway must report the new complete configuration data
+content by generating a new [StatusEvent][message_StatusEvent] message._
+
+#### Get configuration data item
+
+Only processed if CDD_API feature flag is set on the gateway.
+
+- **Request:**
+
+    > **topic:** gw-request/get_configuration_data_item/*\<gw-id\>/\<sink-id\>*
+    >
+    > **content:** [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].GetConfigurationDataItemReq
+
+- **Response:**
+
+    > **topics:** gw-response/get_configuration_data_item/*\<gw-id\>/\<sink-id\>*
+    >
+    > **content:** [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].GetConfigurationDataItemResp
+    >
+    > **possible errors:**
+    > - INVALID_PARAM: No item found with the given endpoint.
```

Comment:
Should the `INVALID_DEST_ENDPOINT` be used here similarly like in the "set" operation to error those cases where the endpoint is invalid (>65536)?